apps/judicial-system/web/src/utils/mutations.ts

```diff
@@ -87,7 +87,7 @@ export const CreateUserMutation = gql`
 `
 
 export const UsersQuery = gql`
-  query Users {
+  query SomeUsers {
```

Comment:
Is this the correct query name?

Reply:
Yes, for now. This is just a temporary name to prevent naming conflicts. This particular query will be removed in later PRs.